src/neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -30,6 +30,7 @@ public class Transaction : IEquatable<Transaction>, IInventory, IInteroperable
         private long sysfee;
         private long netfee;
         private uint validUntilBlock;
+        private UInt256 lastBlockHash = UInt256.Zero;
```

Comment:
If so, why not use `Height` or `SendHeight`, `HeightAt`?

Reply:
Such value can be forged by sender to make duplicate transactions onchain.